src/System.Text.Primitives/System/Text/Encoding/Ascii/PrimitiveEncoder_ascii.cs

```diff
@@ -4,12 +4,12 @@
 
 using System.Runtime.CompilerServices;
 
-namespace System.Text
+namespace System.Text.Encodings
```

Comment:
The other encoders are in System.Text.Encoders right now. Was the usage of "Encodings" intentional?